README.md

```diff
@@ -1,19 +1,21 @@
----
-[DISCLAIMER] Memtest86+ v6.0 is NOT READY FOR PRODUCTION yet. The base code has basically been rewritten from scratch and many side functions are still under active development. A lot of additional beta-testing is needed. Please consider the actual code as experimental and expect crashes and freezes. Bugs reports are welcome and very helpful! Binary beta release are available on [memtest.org](https://memtest.org). The first production-ready stable release is planned for this summer.
-
----
-
 # Memtest86+
 
-Memtest86+ is a stand-alone memory tester for x86 and x86-64 architecture
-computers. It provides a more thorough memory check than that provided by
-BIOS memory tests.
+Memtest86+ is a free, open-source, stand-alone memory tester for x86 and
+x86-64 architecture computers. It provides a much more thorough memory
+check than that provided by BIOS memory tests.
+
+It is also able to access almost all the computer's memory, not being
+restricted by the memory used by the operating system and not depending
+on any underlying software like UEFI libraries.
 
 Memtest86+ can be loaded and run either directly by a PC BIOS (legacy or UEFI)
 or via an intermediate bootloader that supports the Linux 16-bit, 32-bit,
 64-bit, or EFI handover boot protocol. It should work on any Pentium class or
 later 32-bit or 64-bit CPU.
 
+Binary releases (both stable and nightly dev builds) are available on
+[memtest.org](https://memtest.org).
```

Comment:
although the nightly build is still stuck at commit ddbee66 :-(

Reply:
The Windows Server VM used to build the nightly windows USB installer will be back online tomorrow :)

Reply:
Nighty Build server is back online.